src/test/groovy/com/stormpath/tck/util/CookieUtils.groovy

```diff
@@ -22,6 +22,6 @@ class CookieUtils {
 
     public static boolean isCookieDeleted(Cookie cookie) {
         def now = new Date()
-        return cookie.expiryDate < now
+        return cookie == null || cookie.expiryDate < now
```

Comment:
That's true, but this is asserting that the cookie is deleted. If we're expecting a cookie to be actively deleted, it should not be null.